.github/workflows/build.yaml

```diff
@@ -472,3 +511,23 @@ jobs:
             tag=${{ env.BRANCH }}
             dockerRepository=${{ env.REGISTRY }}
             platform=x86_64
+
+  trigger-workflow:
+    name: Trigger workflow depending upon the branch
+    needs: [build-obr, build-obr-generic, build-obr-javadocs]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Triggering helm build (github.ref is main)
+        if: env.BRANCH == 'main'
+        env:
+          GH_TOKEN: ${{ secrets.GALASA_TEAM_GITHUB_TOKEN }}
+        run: |
+         gh workflow run build-helm.yaml --repo https://github.com/galasa-dev/automation --ref ${{ env.BRANCH }}
+        
+      - name: Triggering cli build (github.ref is not main) 
+        if: env.BRANCH != 'main'
+        env:
+           GH_TOKEN: ${{ secrets.GALASA_TEAM_GITHUB_TOKEN }}
+        run: |
+          gh workflow run build.yaml --repo https://github.com/galasa-dev/cli --ref ${{ env.BRANCH }}
```

Comment:
The file in the CLI repo is called `build.yml` - we had to call it that instead of `build.yaml` as there is a rule in the .gitignore that would ignore `build.yaml`. Please can you update the file name?

Reply:
done